homeassistant/components/tuya/cover.py

```diff
@@ -57,7 +38,9 @@ class TuyaCoverEntityDescription(CoverEntityDescription):
     current_state_inverse: bool = False
     current_position: DPCode | tuple[DPCode, ...] | None = None
     set_position: DPCode | None = None
-    control_values: _ControlValues | None = None
+    open_instruction_value: str | None = None
+    close_instruction_value: str | None = None
+    stop_instruction_value: str | None = None
```

Comment:
If we specify the default to be "open", "close" & "stop" here, we can remove it from most locations down below.

Reply:
Have set the default values :)